alignak/objects/resultmodulation.py

```diff
@@ -111,8 +111,6 @@ def pythonize(self):
 
         :return: None
         """
-        # First apply Item pythonize
-        super(Resultmodulation, self).pythonize()
```

Comment:
You are right. Actually we have to remove the whole function. Pythonize is done on properties when instanciating the object. So Item.__init__ is already doing the job.